lib/ruumba/analyzer.rb

```diff
@@ -46,7 +46,10 @@ def analyze(temp_dir, files_or_dirs)
         end
 
       iterator.each do |file, contents|
-        copy_erb_file(file, contents, temp_dir_path)
+        fileFirstLine = File.open(file, &:readline)
```

Comment:
@danielmarzini Thanks for your contribution! This looks reasonable to me. A couple of notes:

1. Prefer `file_first_line` to `fileFirstLine`
2. Prefer `unless` to `if !` here
3. Prefer `'` to `"` in the absence of interpolation
4. Prefer two spaces for indentation

After that, I think we're mergeable 👍

Reply:
tnx, commit on its way.